docs/config.schema.json

```diff
@@ -89,13 +89,46 @@
         "passReferrer": {
           "type": "boolean",
           "description": "Append the referrer URL as a query param on new URL button click"
+        },
+        "isBadge": {
+          "type": "boolean",
+          "description": "Opens the URL in a palette instead of a new tab"
+        },
+        "badgeVariant": {
+          "type": "string",
+          "description": "The variant of the badge following the Adobe Spectrum badge variants",
+          "enum": [
+            "neutral",
+            "informative",
+            "negative",
+            "positive",
+            "notice",
```

Comment:
These semantic colos are mostly aliases for the ones below, so I think we can remove them and just use the non-semantic color names below.